e2e/play-pause.spec.ts

```
@@ -0,0 +1,41 @@
+import { test, expect } from "./fixtures/player.fixture";
+
+test.describe("Play / Pause toggle", () => {
+  test("2-1: clicking play button shows pause icon", async ({ playerPage }) => {
+    const { playBtn, trackCurrentTime } = playerPage;
+    await expect(playBtn).toBeVisible();
+
+    await playBtn.click();
+    await expect(playBtn).toBeVisible();
+
+    // Wait until current time advances past 00:00
+    await expect.poll(() => trackCurrentTime.textContent(), { timeout: 10000 }).not.toBe("00:00");
```

ESLint

[error] Replace `.poll(()·=>·trackCurrentTime.textContent(),·{·timeout:·10000·})` with `⏎······.poll(()·=>·trackCurrentTime.textContent(),·{·timeout:·10000·})⏎······`

(prettier/prettier)